### ISSUE TITLE
Dependency Issue with Debian Buster

### ISSUE THREAD

**Issue by Gill-Bates** [allinurl/goaccess#1629]
On Debian Buster, libssl1.1 is installed

`apt install libssl1.1
Reading package lists... Done
Building dependency tree
Reading state information... Done
libssl1.1 is already the newest version (1.1.1d-0+deb10u2)`

But GoAccess requieres an older version and terminates the installation

`The following packages have unmet dependencies:
 goaccess : Depends: libssl1.0.2 (>= 1.0.2) but it is not installable
E: Unable to correct problems, you have held broken packages.`

Does anyone have any idea how I can persuade GoAccess 1.3 to accept the current version?

**Comment by szepeviktor** [comment]
Same as #1365

**Comment by szepeviktor** [comment]
> Does anyone have any idea how I can persuade GoAccess 1.3 to accept the current version?

```bash
wget -nv -O /root/dist-mod/libssl1.0.2_amd64.deb \
    "http://security.debian.org/debian-security/pool/updates/main/o/openssl1.0/libssl1.0.2_1.0.2u-1~deb9u1_amd64.deb"
dpkg -i /root/dist-mod/libssl1.0.2_amd64.deb
```

**Comment by allinurl** [comment]
This has been fixed and it's now available in [**v1.4**](https://goaccess.io/download). Thank you!
